src/styles/index.scss

```
@@ -1,3 +1,19 @@
+@import './variables';
+@import './card';
+@import './stars';
+
 body {
```

```suggestion
body, h2, p {
```
Consider using commas to group selectors